File: RestaurantUI/src/app/model/class/OrderDTO.ts
import { FoodItem } from "./FoodItem";
import { Restaurant } from "./Restaurant";

export class OrderDTO {
    foodItemsList: FoodItem[];
    userId: number;
    restaurant: Restaurant;

    constructor() {
        this.foodItemsList = [];
        this.userId = 1;
        this.restaurant = new Restaurant();
    }
}

File: order/src/main/java/com/codedecode/order/dto/OrderDTO.java
package com.codedecode.order.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class OrderDTO {

    private Integer orderId;
    private UserDTO userDTO;
    private Restaurant restaurant;
    List<FoodItemDTO> foodItemsList;

}
